go.mod

```diff
@@ -2,26 +2,36 @@ module github.com/ONSdigital/dp-import-api
 
 go 1.17
 
+replace github.com/coreos/etcd => github.com/coreos/etcd v3.3.24+incompatible
```

Comment:
Is it worthwhile to add a comment above on why we are replacing?

Reply:
good idea